src/app/legacy/containers/MediaPlayer/helpers/propsInference/default/index.js

```diff
@@ -29,7 +30,14 @@ export default ({
     path(['model', 'blocks'], aresMediaBlock),
     'caption',
   );
+  const webcastVersions = prop(
+    'webcastVersions',
+    path(['model', 'blocks', 0, 'model'], aresMediaBlock),
+  );
```

Comment:
Can this be written without using Ramda? I feel like part of the problem with it returning different return types was due to the way either Ramda `prop` and/or `path` works, so it may be clearer and easier to debug again if its written in plain JS.